bnb/getBNBData

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+debug = False
+
+import io, os, re, requests, sys, time, zipfile
+from bs4 import BeautifulSoup
+
+maxretry = 3
+datadir = './BNBDaten' if len(sys.argv) < 2 else sys.argv[1]
```

Comment:
Can you document somewhere what the argument(s) are possible?

Reply:
I just copied the behaviour of the old script but will put a description in the script and INSTALL.md.